src/views/additional_info.hbs

```diff
@@ -0,0 +1,20 @@
+{{> header }}
+{{> progress_bar }}
+
+<section class="mb6">
+{{> messages }}
+  <form class="margin-0-auto w-90 w-60-ns w-80-m mw7" action="/additional_info" method="post">
+  <ol class="pa0">
+    <li class="f5 black-60 db mb2 tracked pt3 pb3"><span class='li mr2 w2'>&rarr;</span>
+      Is there anything else you would like to add?
```

Comment:
Can you put this in a label tag? then make the "for" attribute of the label equal to the id of the input element

Reply:
https://www.w3.org/WAI/tutorials/forms/labels/